xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp

```diff
@@ -438,6 +438,10 @@ bool CPVRGUIInfo::GetListItemAndPlayerLabel(const CFileItem* item,
       case LISTITEM_DIRECTOR:
       case LISTITEM_CHANNEL_NUMBER:
       case LISTITEM_PREMIERED:
+      case LISTITEM_PARENTAL_RATING:
+      case LISTITEM_PARENTAL_RATING_CODE:
+      case LISTITEM_MPAA: //For compatibility with other skins.
```

Comment:
Blank after `//`, please.

And what kind of compatibility are we talking about here???